tests/testmthread.py

```diff
@@ -26,7 +26,9 @@
 """
 Tests of the BlackboardItem classes
 """
+from __future__ import print_function
 from __future__ import with_statement
```

Comment:
I think we can remove the `with_statement` item as we no longer have to support python older than 2.7